### ISSUE TITLE
How to apply multi-class instance segmentation with the discriminative loss function?

### ISSUE THREAD

**Issue by HanqingXu** [hq-jiang/instance-segmentation-with-discriminative-loss-tensorflow#8]
Hello, I'm working on a project that needs to detect multiple classes of traffic markings(normal traffic lane marking, arrows, zebra crossing,etc.), so I need to apply a discriminative loss function for multi-class situation where there could still be multiple instancess in each class. I checked your code and as far as I can tell ,it only applies to binary segementation. Any advice about how I can upgrade the loss function such  that it 'll work for multi-class case? Thank you very much
